helm/e2esetup-vault-chart/templates/vault.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
Safe since apps are GA since 1.9; also works as preparation for 1.16 which drops previously deprecated apps/v1beta1 and likes.